packages/extension-browser/src/devtools/views/controls/telemetry-notification.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { useCallback } from 'react';
+
+import * as styles from './telemetry-notification.css';
+
+import Button from './button';
+import ExternalLink from './external-link';
+import Notification from './notification';
+
+import { getMessage } from '../../utils/i18n';
+
+type Props = {
+    show: boolean;
+    onTelemetryChange: (enable?: boolean) => void;
+}
+
+const TelemetryNotification = ({ onTelemetryChange, show }: Props) => {
+    const onEnableTelementry = useCallback(() => {
+        onTelemetryChange(true);
+    }, [onTelemetryChange]);
+
+    const onDisableTelemetry = useCallback(() => {
+        onTelemetryChange(false);
+    }, [onTelemetryChange]);
+
+    const getActions = () => {
```

Comment:
No need for this to be a function. Just store the array of `Button`s here directly and reference it below.
```suggestion
    const actions = ([
```